ca/certificates.go

```diff
@@ -838,9 +850,14 @@ func GetRemoteSignedCertificate(ctx context.Context, csr []byte, rootCAPool *x50
 		}
 
 		// If we're still pending, the issuance failed, or the state is unknown
-		// let's continue trying.
+		// let's continue trying after an exponential backoff
 		expBackoff.Failure(nil, nil)
-		time.Sleep(expBackoff.Proceed(nil))
+		select {
+		case <-ctx.Done():
+			conn.Close(false)
```

Comment:
I'm thinking the parameter to `Close` should be `true`, since it's really `GetRemoteSignedCertificate` that's timing out, and not necessarily anything wrong with the manager. But one could make an argument for keeping it as `false`.